docs/documentation/server_admin/topics/organizations/managing-members.adoc

```diff
@@ -0,0 +1,120 @@
+[id="managing-organization-members_{context}"]
+
+[[_managing_members_]]
+=  Managing members
+[role="_abstract"]
+An organization member is basically a realm user but with a link to a single organization. They are logically separated
```

Comment:
```suggestion
An organization member is basically a realm user but with a link to a single organization. This member is logically separated
```

Reply:
Isn't they being used here as a "neutral" pronoun?

Reply:
Yes but organization member is singular in the preceding sentence.